src/chunk.c

```diff
@@ -2580,15 +2594,37 @@ ts_chunk_drop_chunks(PG_FUNCTION_ARGS)
 				 errmsg("older_than and newer_than timestamps provided to drop_chunks cannot both "
 						"be NULL")));
 
+	if (schema_name == NULL)
+	{
+		/* default schema is public */
+		schema_name = (NameData *) palloc0(NAMEDATALEN);
+		namestrcpy(schema_name, "public");
```

Comment:
the schema shouldnt be hardcoded like that and you should call current_schema to get default schema instead

Reply:
The behavior described in the docs is that we will only look for "public" schema if it is not specified explicitly. We were missing this check earlier.